torch/csrc/jit/ir/alias_analysis.h

```diff
@@ -171,6 +171,7 @@ class AliasDb {
   void enablePreciseTupleContainerAnalysis();
 
   friend struct MutationRemover;
+  friend class TensorExprFuser;
```

Comment:
There is no fundamental reason for TE Fuser to be relying on private details of AliasDB implementation.